src/Gui/InputField.cpp

```diff
@@ -699,6 +704,9 @@ void InputField::keyPressEvent(QKeyEvent *event)
 
 void InputField::wheelEvent (QWheelEvent * event)
 {
+    if (!(hasFocus() || (focusPolicy() & 0x4)))

```

Comment:
What's the magic number?

Reply:
Because it doesn't work. :) 

However let me have another look. Maybe using focus is enough that will make things more readable.

Reply:
OK, I checked and `hasFocus()` is enough. ;)